cogs/utility.py

```diff
@@ -827,6 +830,25 @@ def get_datetime():
                 await guild.ack()
             await ctx.send(self.bot.bot_prefix + "Marked {} guilds as read.".format(len(self.bot.guilds)))
             
+    @commands.command(aliases=['hu','hupdate'])
+    async def herokuupdate(self, ctx):
+        '''Auto Update command, checks if you have latest version'''
+        git = self.bot.get_cog('Git')
```

Comment:
This could potentially conflict with the very common custom cog by @GriffinG1 (https://raw.githubusercontent.com/GriffinG1/Custom-Cogs/master/git.py), could you find a different way of doing this?